blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -38,10 +38,13 @@ Unreleased Changes
 .. contents::
    :local:
 
+
 Changes
 =======
 
-None
+- Changed the default for :ref:`sql_ref_write_wait_for_active_shards` from
+  ``ALL`` to ``1``.
```

Comment:
Shall we maybe explain the decision, or highlight that this lowers write resiliency in favour of write availability (not sure if this is the right wording)?

Reply:
I'll extend the current settings description in a separate PR and then also address this